cuizinart/frontend/src/index.js

```diff
@@ -216,10 +217,19 @@ class App extends Component {
                 }
             })
             .catch(function (error) {
-                NotificationManager.error(error.message);
+                let message = '';
+                if (error.response && error.response.data && error.response.data.response
+                    && error.response.data.response.errors){
+                    let err = error.response.data.response.errors;
+                    message = err[Object.keys(err)[0]][0];
+                }
+                else {
+                    message = error.message;
+                }
+                NotificationManager.error(message);
```

Comment:
This handling is always the same, right? Can we encapsulate the extraction of `message`?